rdrf/rdrf/models/definition/models.py

```diff
@@ -1946,3 +1946,14 @@ def __str__(self):
 
     class Meta:
         verbose_name = "Disallowed mime type"
+
+
+class DeviceCookie(models.Model):
+    user = models.ForeignKey('groups.CustomUser', null=False, blank=False, on_delete=models.CASCADE)
+    cookie = models.TextField()
+    max_age = models.PositiveIntegerField(null=False, blank=False)
```

Comment:
`null=False, blank=False` omit please as they are the default. also in next line. And on `user` above